lib/TfIdfVectorizer.jsm

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+
+// Javascript doesn't support unicode page mnemonics out of the box, and so
+// we have to define them ourselves.
+const UNICODE_SPACE = "\x20\xA0\u1680\u2000-\u200A\u2028\u2029\u202F\u205F\u3000";
```

Comment:
Please add the comments that you added in 6eae5458e65458e17b8e4d7c003f0cfb6cca7ff5?